openslides_backend/action/actions/user/forget_password.py

```diff
@@ -0,0 +1,136 @@
+from collections import defaultdict
+from smtplib import (
+    SMTPAuthenticationError,
+    SMTPDataError,
+    SMTPRecipientsRefused,
+    SMTPSenderRefused,
+    SMTPServerDisconnected,
+)
+from ssl import SSLCertVerificationError
+from time import time
+from typing import Any, Dict
+
+from ....models.models import User
+from ....shared.exceptions import ActionException
+from ....shared.filters import FilterOperator
+from ...generics.update import UpdateAction
+from ...mixins.send_email_mixin import EmailMixin, EmailSettings
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData
+
+PW_FORGET_EMAIL_TEMPLATE = """You receive this email, because you have requested a new password for your OpenSlides-account.
+
+Please open the following link and choose a new password:
+/login/forget-password-confirm?user_id={user_id}&token={token}
```

Comment:
Mmh, without a host this link will obviously not work. Maybe we need to do the work and make the host available in the backend? @GabrielInTheWorld @gsiv

Reply:
Yes, I agree. At some time, we have to do this work anyway - therefore, we can do it now.

Reply:
Ok, we'll do that in a new issue.